app/lib/audit.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_record_utils'
+require 'druid-tools'
+require 'profiler'
+require 'csv'
+
+require_relative 'audit/catalog_to_moab'
+require_relative 'audit/checksum'
+require_relative 'audit/moab_to_catalog'
+
+# @example Usage
+#  require 'audit'
+#  ...
+#  Audit::Checksum.validate_disk(storage_root)
+module Audit
```

Comment:
why do we need an empty `module Audit` here?

Reply:
Just convention to define the namespace.  Not strictly necessary, and I'd remove it if ppl prefer.